administration/src/util/normalizeString.ts

```diff
@@ -1,6 +1,11 @@
-import normalizeStrings from 'normalize-strings'
+// eslint-disable-next-line no-control-regex
```

Comment:
:question: Could you explain why this needs to be disabled/why you are checking for this?

Reply:
Because the rule wants to forbid mentioning control characters in JS regex'es.
However, the ASCII symbols have control characters :D
Seems like a weird rule to me, but yeah :D